rosbag2_test_common/include/rosbag2_test_common/bag_filename_for_storage_id.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022, Foxglove Technologies Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ROSBAG2_TEST_COMMON__BAG_FILENAME_FOR_STORAGE_ID_HPP_
+#define ROSBAG2_TEST_COMMON__BAG_FILENAME_FOR_STORAGE_ID_HPP_
+#include <vector>
+#include <string>
+
+
+namespace rosbag2_test_common
+{
+std::string bag_filename_for_storage_id(const std::string & name, const std::string & storage_id)
```

Comment:
I think delete this file and include this in the `tested_storage_ids`, remove duplicated plugin name data.